src/lib/gundb/UserStorageClass.js

```diff
@@ -1354,7 +1290,7 @@ export class UserStorage {
    * @returns {object} UserModel with some inherit functions
    */
   getPrivateProfile(profile: {}): Promise<UserModel> {
-    const keys = Object.keys(profile)
+    const keys = Object.keys(profile).filter(k => ['initialized'].includes(k) === false)
```

Comment:
```suggestion
    const keys = Object.keys(profile).filter(key => !['_', 'initialized'].includes(key))
    
```

Reply:
also this filtering is used here: https://github.com/GoodDollar/GoodDAPP/blob/master/src/lib/gundb/UserStorageClass.js#L2559

so we could add an private method for that to be reused:

```javascript
/**
 * @private
 */
_getProfileFields = profile => filter(keys(profile), field => !['_', 'initialized'].includes(field))
```

Reply:
done